tools/jenkins-slave-creation-unix/conf-ubuntu-gpu/install.sh

```diff
@@ -157,6 +156,9 @@ echo 'mxnet-linux-cpu10' > /home/jenkins_slave/jenkins_slave_name
 "
 echo "$readme" > /home/ubuntu/readme.txt
 
+# Disable automatic Ubuntu updates
```

Comment:
I understand our instances typically don't run very long and automatic updates take do not get installed immediately at start (so there is always a time where the instance is unpatched). But currently the instance will apply security updates, whereas after this change, the instance will always be unpatched.

If we like to disable automatic updates, we should have a mechanism to redeploy a patched AMI on a regular basis.

Reply:
Alright, makes sense. Removed the disable part & re-deployed the AMI.